libraries/AP_Arming/AP_Arming.h

```diff
@@ -141,6 +141,7 @@ class AP_Arming {
     AP_Float                accel_error_threshold;
     AP_Int8                 _rudder_arming;
     AP_Int32                _required_mission_items;
+    AP_Int16                _max_arm_dist_m; //maximum allowed distance to first waypoint (m)
```

Comment:
needs to be AP_Float, 32km is too small as a max

Reply:
I don't know much about plane operation, but is 32km too small even for just the first waypoint?

Reply:
Yes